AUTHORS

```diff
@@ -39,4 +39,3 @@ Dean Elbaz <elbaz.dean@gmail.com>
 Mike Berman <evencode@gmail.com>
 Dmitriy Fedorenko <c0va23@gmail.com>
 Zach Marcantel <zmarcantel@gmail.com>
-James Maloney <jamessagan@gmail.com>
```

Comment:
Sorry, I don't understand what you mean. @james-maloney submitted #291, so I'm not sure that his name should be removed from the `AUTHORS` list.

Also, in the context of #291, this #287 is effectively undoing the change from #291 - what is the intention behind moving the

```go
if v, ok := value.(Marshaler); ok {
    return v.MarshalCQL(info)
}
```

below the 

```go
if valueRef := reflect.ValueOf(value); valueRef.Kind() == reflect.Ptr {
    if valueRef.IsNil() {
        return nil, nil
    } else if _, ok := value.(Marshaler);  !ok {
        return Marshal(info, valueRef.Elem().Interface())
    }
}
```

block?